client/src/app/my-projects/layout.tsx

```diff
@@ -0,0 +1,17 @@
+"use client";
+
+import { PropsWithChildren } from "react";
+
+import { MapProvider } from "react-map-gl";
+
+import { LayoutGroup } from "framer-motion";
+
+export default function MyProjectsLayout({ children }: PropsWithChildren) {
+  return (
+    <MapProvider>
```

Comment:
You don't need this provider, it's only used in pages where we have a Map implemented, and that only happens in the overview page.

Reply:
Yes you are right, it does work without the layout file as we have that in common. Thanks !